pkg/scheduler/core/equivalence/eqivalence.go

```diff
@@ -282,6 +291,7 @@ func (n *NodeCache) lookupResult(
 	n.mu.RLock()
 	defer n.mu.RUnlock()
 	value, ok = n.cache[predicateKey][equivalenceHash]
+	delete(n.receivedInvalidatioRequests, predicateKey)
```

Comment:
It seems like anybody who does a lookup will clear the flag for that predicate, causing the next updater to assume that the cache has not been invalidated.